assets/app/features/upgrade/templates/landing-page_step3.partial.jade

```diff
@@ -0,0 +1,12 @@
+.upgrade(ng-class="{'active': upgradeLandingVm.mode.valid}")
+    .step-title.clearfix
+        crowbar-circle.pull-left(ng-class="{'active': upgradeLandingVm.mode.valid}") 3
+        h2.h4(translate='') upgrade.steps.landing.upgrade.title
+    p(translate='') upgrade.steps.landing.upgrade.description
```

Comment:
Visually I think there should be some more free space between the steps (i.e. above `(2)`) and less above "Begin Upgrade" button (see: https://raw.githubusercontent.com/crowbar/crowbar-ui/wiki-assets/upgrade67/landing%20page2.png).